index.html

```diff
@@ -81,6 +81,167 @@ <h5 class="sponsor-heading">sponsored by</h5>
             </ul>
           </div>
           <div class="col-sm-12">
+            <h2>Schedule</h2>
+            <h3> Day 1 </h3>
+            <table class="table table-bordered schedule">
+              <thead>
+                <tr>
+                  <th></th>
+                  <th class="room"><h3>Main Room</h3></th>
+                  <th class="room"><h3>Second Room</h3></th>
+                  <th class="room"><h3>Workshop Room</h3></th>
+                </tr>
+              </thead>
+              <tbody>
+              <tr>
+                <th scope="row">8:00</th>
+                <td colspan="2" class="red">Registration</td>
+              </tr>
+              <tr>
+                <th scope="row">9:00</th>
```

Comment:
I think @haard was keen on not talking 30 minutes, so set the opening to 9:15 or 9:20? :)